recipes/xraylarch/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "xraylarch" %}
+{% set version = "0.9.50" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/xraypy/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: f64eee63bca13a67d313a06c9b437502ea59a5ca961075756ebf453e9dc1da81
+  patches:
+    - 0001-MNT-remove-the-desktop-shortcut-logic.patch
+    - 0002-MNT-remove-the-feff-wrappers-and-binaries.patch
+
+build:
+  number: 0
+  script: |
+    # We do not have a license to distribute built FEFF binaries.
```

Comment:
LGTM but I'm not an expert on the topic. @isuruf do you think that removing the binaries "covers" it with regarding to license to re-distribute/package this?

Reply:
We can try to remove the patch, I think it's OK now without it.